### PR TITLE
Added new product card

### PR DESCRIPTION
**Pull request by julia-queen**
## Done

- Added a new product card _ubuntu_18.04_end_of_support for pages with the tag "ubuntu 18.04"

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by tbille** [comment]
@julia-queen I am not really familiar with this part of the site. Where should I look for in demo: https://ubuntu-com-12718.demos.haus/

**Comment by julia-queen** [comment]
Hi @tbille ! Any blockers on this request?

File: templates/blog/product-cards/_ubuntu_18.04_end_of_support.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="p-card js-product-card u-hide">
+    <h3>
+      <a href="/18-04-end-of-standard-support">Time to prepare for Ubuntu 18.04 LTS End of Standard Support</a>
```
Comment (caldav) [inline templates/blog/product-cards/_ubuntu_18.04_end_of_support.html:3]:
```suggestion
      <a href="/18-04">Time to prepare for Ubuntu 18.04 LTS End of Standard Support</a>
```

File: templates/blog/product-cards/_ubuntu_18.04_end_of_support.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    </h3>
+    <p>Bionic Beaver will reach the end of the standard, five-year maintenance window for Long-Term Support (LTS) releases on 31 May 2023. Migrate to the latest LTS or get extended coverage until 2028 with Ubuntu Pro.</p>
+    <p><a href="/18-04-end-of-standard-support">Learn more&nbsp;&rsaquo;</a></p>
```
Comment (caldav) [inline templates/blog/product-cards/_ubuntu_18.04_end_of_support.html:6]:
```suggestion
    <p><a href="/18-04">Learn more&nbsp;&rsaquo;</a></p>
```

**Review by caldav — CHANGES_REQUESTED** [review]
Proposed a new link since we now have the /18-04 page live. Other than that looking good! Will approve once the suggestions are applied.

**Review by caldav — APPROVED** [review]
:+1:

**Pull request merged**